runtime/jailer_integ_test.go

```diff
@@ -141,7 +141,8 @@ func testJailer(t *testing.T, jailerConfig *proto.JailerConfig) {
 	stdout := startAndWaitTask(ctx, t, c)
 	require.Equal("hello\nadditional drive\n", stdout)
 
-	stat, err := os.Stat(filepath.Join(shimBaseDir(), "default", vmID))
+	nsp := "default-" + vmID
```

Comment:
here and everywhere else -- is there a `getter` that we could use?

Reply:
I don't think it is necessary to create a getter here. I move the namespace into oneline.